crates/types/src/v1alpha1/wasmcloud_host_config.rs

```diff
@@ -39,7 +39,7 @@ pub struct WasmCloudHostConfigSpec {
     pub nats_leaf_image: Option<String>,
     /// The name of a secret containing the primary cluster issuer key along with an optional set
     /// of NATS credentials.
-    pub secret_name: String,
+    pub secret_name: Option<String>,
```

Comment:
I think it's worth updating the comment above to note that this secret is expected to contain a set of NATS Credentials if it's set